test/Executor.t.sol

```diff
@@ -172,27 +169,24 @@ contract ExecutorConstructorTests is ExecutorTestBase {
         emit DelayUpdate(0, DELAY);
```

Comment:
Lets add address to expectEmits here

Reply:
I don't think it's relevant here because the address is not live until it is constructed. Would have to do a vm.computeAddress to get the next address.